ExtensionBundle.Tests/DependencyValidationTests.cs

```diff
@@ -0,0 +1,150 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Build;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Microsoft.Extensions.DependencyModel;
+using Xunit;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Azure.Functions.ExtensionBundle.Tests
+{
+    public class DependencyValidationTests
+    {
+        private readonly DependencyContextJsonReader _reader = new DependencyContextJsonReader();
+        private readonly IEnumerable<string> _rids = DependencyHelper.GetRuntimeFallbacks();
+
+        [Fact]
+        public void Verify_DepsJsonChanges()
+        {
+            string oldDepsJson = Path.GetFullPath("../../../TestData/windows_extensions.deps.json");
```

Comment:
code for reference - [Link](https://github.com/Azure/azure-functions-host/blob/b612457d32902cd4b6b9c5b32b5eecb20240549c/test/WebJobs.Script.Tests/DependencyTests.cs#L42)